build.libraries.gradle

```diff
@@ -1,22 +1,22 @@
 ext {
-    springBootVersion = System.getenv("SPRING_BOOT_VERSION") ?: "2.6.14"
+    springBootVersion = System.getenv("SPRING_BOOT_VERSION") ?: "3.0.6"
 
     libraries = [
             // explicit versions
+            hibernateCoreJakarta            : 'org.hibernate:hibernate-core-jakarta:5.6.15.Final',
```

Comment:
Spring Boot 3 is using Hibernate 6, which is already using jakarta packages, so we can keep using hibernate-core and Spring Boot 3 dependencies BoM will control the version itself.